### ISSUE TITLE
Search: enable debug bar support for ES queries

### ISSUE THREAD

**Issue by gibrown** [Automattic/jetpack#8564]
We have some debug bar support on wp.com. That should be ported to work with the JP Search module.

Should make sure that it works well in both https://wordpress.org/plugins/debug-bar/ and https://wordpress.org/plugins/query-monitor/

**Comment by gibrown** [comment]
Can probably use a filter like this.

```
add_filter( 'jetpack_search_es_query_args', function( $es_query_args, $query ) {
	if ( ! is_user_logged_in() ) {
		//return $es_query_args;
	}

	echo '<div style="background-color:white;padding:10px;">';
	var_dump( $es_query_args );
	echo '</div>';

	return $es_query_args;
}, 10, 2 );
```

**Comment by ebinnion** [comment]
Hey @gibrown – I'm not sure I follow what that filter is for?

**Comment by Viper007Bond** [comment]
@ebinnion He was using it as an example for how to snag the ES query args so that they can be displayed in the debug bar.

**Comment by ebinnion** [comment]
ahh, thank you sir 🙇

**Comment by ebinnion** [comment]
Addressed with #8615
